deploy/api.yaml

```diff
@@ -2,7 +2,53 @@ openapi: "3.0.1"
 info:
   version: "0.1"
   title: "Address Credential Issuer API"
+
+x-amazon-apigateway-request-validators:
+  body:
```

Comment:
Renaming this validator to body breaks the request validator for the /session API which is expecting a "Validate body" validator